docs/contributing/gsoc-ideas.md

```diff
@@ -588,7 +587,7 @@ Where should you publish your draft? We prefer Dropbox Paper or
 Google Docs, since those platforms allow people to look at the text
 without having to log in or download a particular app, and you can
 update the draft as you improve your idea. In either case, you should
-post the draft for feedback in chat.zulip.org.
+post the draft for feedback in [The Zulip developer community](https://zulip.com/developer-community/).
```

Comment:
I would recommend wrapping all lines at around 80 characters just like the surrounding text.

Reply:
@eeshangarg do want me to do it in this way?
![ezgif com-gif-maker](https://user-images.githubusercontent.com/76884959/135060517-25cac766-7a30-489a-aa17-4252f27170a7.gif)

Reply:
I think the entire link's text would also have to move to the next line. You cannot break Markdown links over to the next line. For long links, use numbered references, see [this doc](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) for an example.